packages/lesswrong/components/questions/NewQuestionDialog.jsx

```diff
@@ -75,7 +75,7 @@ class NewQuestionDialog extends PureComponent {
         <DialogContent>
           <Components.WrappedSmartForm
             collection={Posts}
-            fields={['title', 'content', 'body', 'question', 'draft']}
+            fields={['title', 'contents', 'body', 'question', 'draft']}
```

Comment:
!!? What does this do? Is `NewQuestionDialog` broken in production by this having been incorrect?

Reply:
Yes, NewQuestionsDialog does not currently work